examples/09_Kotlin_JS/01_dynamic.md

```diff
@@ -30,9 +30,9 @@ fun main(){
 
 </div>
 
-1. Any value can be assigned to variable to `dynamic` type
-2. Dynamic value can be assigned to anything.
-3. Dynamic value can be passed as argument to any function
-4. Any property or function with any arguments can be called on `dynamic` value 
-5. Dynamic call always returns dynamic value, so it is possible to chain them.
-6. Operators, assignments and indexed access (`[..]`) are translated "as is". Beware!
+1. Any value can be assigned to variable of the `dynamic` type.
```

Comment:
Any value can be assigned to a `dynamic` variable type.

--is this right?

Reply:
Yes, type checks aren't performed for dynamic.